google/ggrc-core
# GGRC-6468 Create 'tree-item-attr' component

# Issue description

Currently there are several similar templates for different object types. Path to appropriate template been passed to 'tree-item' component and after that is rendered on the fly.  

# Steps to test the changes

 - Open any tab w/ tree-view
 - Expand 'visible-columns-list' dropdown
 - Select some options

Expected behavior: Selected options are displayed correctly for each tree item element.

# Solution description

Create 'tree-item-attr' component w/ template that includes options from all existing templates. Add appropriate tag to 'tree-item' & 'mapper-results-item-attr' templates.

# Sanity checklist




# PR Review checklist
